pytools/graph.py

```diff
@@ -178,63 +179,85 @@ def __init__(self, node):
         self.node = node
 
 
-def compute_topological_order(graph):
-    """Compute a toplogical order of nodes in a directed graph.
+class HeapEntry:
+    def __init__(self, node, key):
+        self.node = node
+        self.key = key
+
+    def __lt__(self, other):
+        return self.key < other.key
```

Comment:
Could you add a comment pointing to docs that that's all that's needed for `heapq`? (Aka why doesn't this need the full assortment of `__eq__`, `__le__`, etc?)

Reply:
Didn't find any mentions for it in the docs. However the code for `heapq` says it. See https://github.com/python/cpython/blob/8d21aa21f2cbc6d50aab3f420bb23be1d081dac4/Lib/heapq.py#L135-L138

Reply:
Didn't find any mentions for it in the docs. However the code for `heapq` says it. See https://github.com/python/cpython/blob/8d21aa21f2cbc6d50aab3f420bb23be1d081dac4/Lib/heapq.py#L135-L138